src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -115,7 +115,7 @@ public async Task<IActionResult> Index(ContentOptions options, PagerParameters p
                 new SelectListItem(S["Delete"], nameof(ContentsBulkAction.Remove)),
             ];
 
-            return View(model);
+            return View("Index", model);
```

Comment:
This isn't needed. By default View(model) calls a view with the same name as the action.

Reply:
I see. Then maybe we should add a document explaining the reason for needing in in the code. Also change please `"Index"` to `nameof(Index)`

Reply:
Added comment that describes the reason it's needed in this case.